src/components/Search.js

```diff
@@ -39,10 +40,14 @@ export const Search = () => {
     }
   }
 
-  const goToRandomArticle = () => {
-    getRandomArticleTitle(lang).then(title => {
-      goto.article(lang, title)
-    })
+  // const goToRandomArticle = () => {
```

Comment:
We've discussed we don't want to remove the random article feature altogether, there's a ticket already ([T248124](https://phabricator.wikimedia.org/T248124)) to decide where we should move it to for testing purposes. 

On that note, I think once we merge this PR we will have enough to mark ([T245156](https://phabricator.wikimedia.org/T245156)) as resolved. 

To be decided if we leave these respective lines commented out in this PR or deleted

Reply:
Decided during daily meeting to just remove random article feature separately: https://github.com/wikimedia/wikipedia-kaios/pull/189